src/Pagination.tsx

```diff
@@ -28,14 +28,6 @@ const propTypes = {
   size: PropTypes.oneOf<PaginationSize>(['sm', 'lg']),
 };
 
-/**
- * @property {PageItem} Item
- * @property {PageItem} First
- * @property {PageItem} Prev
- * @property {PageItem} Ellipsis
- * @property {PageItem} Next
- * @property {PageItem} Last
- */
```

Comment:
not sure why those are removed

Reply:
I didn't think these were used (at least in the docs perspective), and not all our components with additional properties was documented like Modal or Form.  Is it needed?

Reply:
i don't think they are - it's just jsdoc anyways :smile: